build.gradle

```diff
@@ -24,6 +24,7 @@ ext {
     aggregatorVersion='1.2.1'
     filtersVersion='1.6.1'
     eventClientVersion='1.5.1'
+    eventClientVersion='1.5.2'
```

Comment:
Make this 1.6 instead?

Reply:
Yes, code change do the minor version, dependabot changes, do patch version.